### PR TITLE
Add presenting slides actions to media session article

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR adds presenting slides actions to the media session API article. This article needs to be published when ~~Chrome 110 reaches Beta channel (January 12nd, 2023)~~ Chrome 111 reaches Beta channel (Thu, Feb 9, 2023)  

Live preview: https://deploy-preview-9210--web-dev-staging.netlify.app/media-session/#presenting-slides-actions

See https://chromestatus.com/feature/5090980649238528

### CODE REVIEW

File: src/site/content/en/blog/media-session/index.md
Comment (rachelandrew) [inline src/site/content/en/blog/media-session/index.md:489]: Please use the compat component rather than a note that will go out of date.

**Review by rachelandrew — CHANGES_REQUESTED** [review]
See note inline, if BCD does not have this data then we need to add it.

File: src/site/content/en/blog/media-session/index.md
Comment (beaufortfrancois) [reply src/site/content/en/blog/media-session/index.md:489]: https://developer.mozilla.org/en-US/docs/Web/API/MediaSession/setActionHandler and https://github.com/search?q=repo%3Amdn%2Fbrowser-compat-data%20setActionHandler&type=code show you that MDN doesn't have this kind of granularity yet, hence the note for now.

Comment (rachelandrew) [reply src/site/content/en/blog/media-session/index.md:489]: It just needs adding to BCD.

Comment (rachelandrew) [reply src/site/content/en/blog/media-session/index.md:489]: If MDN has no docs for this please add it to the API docs trix and note that BCD doesn't yet exist.

Comment (beaufortfrancois) [reply src/site/content/en/blog/media-session/index.md:489]: @rachelandrew It's not clear to me if this PR is now ready to be merged or if I need to take some other action.

**Comment by rachelandrew** [comment]
@beaufortfrancois no, because the feature isn't in Chrome yet. Hopefully we can get the BCD updated before it's merged to (as discussed above).

**Comment by beaufortfrancois** [comment]
IIUC If BCD are not updated **before** Chrome 110 goes Beta, we'll merge this post as it and update it after. Is that accurate?

**Comment by rachelandrew** [comment]
We'll merge it when it's in stable, this is web.dev not d.c.c. and we need to make sure it has clear compat information at that point so it's not confusing to developers as this is a change to an existing feature that has good support. If you want a beta post about this, write a short blog post on d.c.c. and we can link to it from the beta release notes.

**Comment by rachelandrew** [comment]
@beaufortfrancois great! For it to show up, it needs to be in the release version of the BCD package, which I will then update here. I'll keep an eye out for it.

**Comment by beaufortfrancois** [comment]
FYI media session slides actions will be enabled by default in Chrome 111, not Chrome 110. See https://github.com/mdn/browser-compat-data/pull/18463

**Comment by beaufortfrancois** [comment]
@rachelandrew  https://developer.mozilla.org/en-US/docs/Web/API/MediaSession/setActionHandler#browser_compatibility is now updated with Chrome 111 data. I think we can merge this CL.

**Comment by beaufortfrancois** [comment]
@rachelandrew Chrome 111 is now stable. Shall we merge this PR?

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**
